spec/spec_helper.rb

```diff
@@ -56,7 +56,11 @@
   config.filter_run_when_matching :focus
 
   config.before(:each) do
-    Rack::Attack.enabled = false
+    begin
+      Rack::Attack.enabled = false
+    rescue
+      # RackAttack not loaded - probably running a single test
```

Comment:
did you mean to leave this in here? if so is it a normal pattern to have a rescue block with just a comment?

Reply:
When running `rspec spec/i18n_spec.rb` I was getting:
```
Failure/Error: Rack::Attack.enabled = false
     
     NameError:
       uninitialized constant Rack
       ...
```

Although swallowing exceptions is frowned upon, we don't need to do anything if it RackAttack not been required - it just means we are running a single spec. I have updated this to print a warning message just in case.